code/__DEFINES/materials.dm

```diff
@@ -9,6 +9,8 @@
 
 /// Used to make a material initialize at roundstart.
 #define MATERIAL_INIT_MAPLOAD	(1<<0)
+/// Used to make a material init bespoke.
+#define MATERIAL_INIT_BESPOKE	(1<<1)
```

Comment:
This does not explain what being bespoke is to people who read the comment, maybe elaborate a bit more.

Reply:
Done